assets/src/settings/index.js

```diff
@@ -56,50 +66,51 @@ class Settings extends Component {
 		} = this.state;
 
 		return (
-			<div className="ncc-page">
-				<h1>{ __( 'Content Conversion Settings' ) }</h1>
-				<br />
-
-				<p>
-					{ __(
-						'Adding content to the queue, enables it to be converted to Gutenberg Blocks. The queue is also a backup point for possible reverting.'
-					) }
-				</p>
-				<p>
-					{ __(
-						'Existing HTML content is first selected by type, and added to a conversion queue. Queued content is then converted.'
-					) }
-				</p>
-				<h3>{ __( 'Specify content type' ) }</h3>
-				<ul>
-					<li>
-						content types:{' '}
-						<input type="text" disabled={ true } value={ conversionContentTypesCsv } />
-					</li>
-					<li>
-						content statuses:{' '}
-						<input type="text" disabled={ true } value={ conversionContentStatusesCsv } />
-					</li>
-				</ul>
-
-				<h3>{ __( 'Conversion params' ) }</h3>
-				<ul>
-					<li>
-						conversion batch size:{' '}
-						<input type="text" disabled={ true } value={ conversionBatchSize } />
-					</li>
-				</ul>
-
-				<h3>{ __( 'Queued stats' ) }</h3>
-				<ul>
-					<li>
-						<b>
-							<u>{ queuedEntries }</u>
-						</b>{' '}
-						posts are currently selected and queued for conversion
-					</li>
-				</ul>
-			</div>
+			<Fragment>
+				<div className="newspack-logo-wrapper">
+					<NewspackLogo />
+				</div>
+				<Grid>
+					<FormattedHeader
+						headerIcon={ <SettingsIcon /> }
+						headerText={ __( 'Content conversion settings' ) }
+						subHeaderText={ __( 'Adding content to the queue to convert it to Gutenberg Blocks.' ) }
+					/>
+					<Card>
+						<p>
+							{ __(
+								'Existing HTML content is first selected by type, and added to a conversion queue. Queued content is then converted. The queue is also a backup point for possible reverting.'
```

Comment:
Proposing change in text:
The type of HTML content to be converted to Gutenberg blocks is specified here. This content is then added to a "conversion queue". This queue is also a backup point for possible reverting. To start the conversion, go to the "Run Conversion" page (link to '/wp-admin/admin.php?page=ncc-conversion').

Reply:
See fa48435.

I actually added a button instead of the sentence `To start the conversion, go to the "Run Conversion" page`.